me.broken_by.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,32 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: pmedit-cli
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v app/* ${FLATPAK_DEST}
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
```

Comment:
```suggestion
```

My suggestion was also to remove these.

Reply:
I see. I am sorry I kind of glazed over that.